src/validation/rules/__tests__/recommendedModelBodyFieldPostfix.rule.test.ts

```diff
@@ -23,6 +23,26 @@ describe(__filename, () => {
 `);
   });
 
+  it('should be valid v2', () => {
+    expectValid(`
+    RequestQuery {
+        body,
+    }
+      
+    RequestQueryRequestBody {}
+`);
+  });
+
+  it('should be valid v3', () => {
```

Comment:
it's a shitty naming for tests, meh. Maybe it should express something meaningful?